Doom-Mummies/Welcome-to-Nashville
# added files and structure for stretch goals

# Description

added a database.json file in the API folder that is being ignored. Also create a navbar.js and a json.js file for stretch goals.


Please delete options that are not relevant.



# Testing Instructions
1. `git fetch --all`
2. `git checkout NAMEOFBRANCH`
3. In terminal, from src type `tree` to see new files.
4. type `code .` to open files.

# Checklist:
